src/utils/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
```

Comment:
Why is this placed in utils? Shouldn't it be in inputs like all other buttons are?

Reply:
I see it more like and utility than input, but I don't have strong opinions about it

Reply:
Although all the components inside the input category should be used inside a form. This is not the case.

Reply:
So we shouldn’t use the button from `inputs` for triggering scripts? Can we point this in a documentation?

Reply:
I think the structure of the components should be refactored, could use evergreen as an example: https://evergreen.segment.com/components

Reply:
> So we shouldn’t use the button from inputs for triggering scripts? Can we point this in a documentation?

I think we should, what I want to say is that input category is more related (not exclusive) to inputs form. As I user of the lib, if I want to search for a copy button, I would search first in utilities before inputs. But again, just my opinion.

Reply:
> I think the structure of the components should be refactored, could use evergreen as an example: https://evergreen.segment.com/components

We discussed this before and we decided to follow the material-ui structure. but sure, we can change that, please create an issue and we can reorganize the categories.